### DISCUSSION TITLE
Riggify availability

### DISCUSSION THREAD

**Discussion by Electricavisuals** [mrachinskiy/commotion#32] [Ideas]
Hi I am trying to animate a long fish, and my rig is made in riggify... so, 
In Pose mode, we loose Commotion tab on de N menu... 
it will be useful to animate long undulating tails, snakes, fishes and other beings.... 
Thank you

**Comment by mrachinskiy** [comment]
Hi, the reason it is not available in pose mode, is because it is not possible to just copy action with all f-curves at once, every keyframe should be recreated one by one for each bone. Considering that drivers and f-curve modifiers also need to be recreated, I decided to postpone this feature until I have time to do it, or until there will be a better API for this task.
